cmd/operator-ui/Gopkg.lock

```diff
@@ -53,11 +65,12 @@
     "pkg/util/error",
   ]
   pruneopts = "UT"
-  revision = "370ac9fd2f493ce9aa8fdc57ad6a67dcb4458e62"
+  revision = "468dd34a4629594652ddb719c536d85e4ba8e83b"
+  version = "0.1.1"
 
 [[projects]]
   branch = "release-2019.4.x"
```

Comment:
@jdartigalongue For master branch of operator-ui, the synopsys-operator should be master right?